### ISSUE TITLE
Specifying testRunnerCodeunitId is not supported when using the C/AL test runner (BC14)

### ISSUE THREAD

**Issue by BernhardReu** [jimmymcp/al-test-runner#63]
Version: 0.7.7

When executing a test for BC14 with the newest version of al-test-runner it throws the following error:
`Specifying testRunnerCodeunitId is not supported when using the C/AL test runner`

It has been working great so far with previous versions, and after rolling back to 0.6.3 I am able to execute the test(s) again.

I looked at the recent changes and guess it could have something to do with the new default value of the `TestRunnerCodeunitId` in `Invoke-RunTests.ps1` in combination with the fact that the paramater is only added if it is greater 0 in the config at `Invoke-ALTestRunner.ps1`:
```
    if ((Get-ValueFromALTestRunnerConfig -KeyName 'testRunnerCodeunitId') -gt 0) {
        $Params.Add('TestRunnerCodeunitId', (Get-ValueFromALTestRunnerConfig -KeyName 'testRunnerCodeunitId'))
    }
```

**Comment by jimmymcp** [comment]
Thanks, should be fixed in 0.7.8 - deploying at the moment...

**Comment by BernhardReu** [comment]
Wow, that was quick, thank you! 
And thanks for making TDD for BC development possible with this extension.
